### ISSUE TITLE
Add. 14,530 | https://bl.vuexistapps.us/manuscript/257

### ISSUE THREAD

**Issue by ZiyanZhang13** [srophe/britishLibrary-data#405]
### Problems with Manuscript Overview

Missing the type of writing, should be "Unspecified Syriac"

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

**Comment by haeleekim** [comment]
should the end of the physDesc go into additions?; or mscontents

**Comment by wlpotter** [comment]
Agreed that this should be moved out of the physDesc. Let's put the whole sentence ("From a note on the margin...bound with this manuscript") into an addition item. Keep the quote element, but we don't need the title element.

This is another case where we'll want some of that info to go into a provenance element, so once the change here is made leave this issue open and we'll follow up
